packages/block-library/src/cover/edit.js

```diff
@@ -329,32 +393,26 @@ class CoverEdit extends Component {
 		return (
 			<>
 				{ controls }
-				<ResizableBox
+				<ResizableCover
 					className={ classnames(
 						'block-library-cover__resize-container',
 						{ 'is-selected': isSelected },
 					) }
-					size={ {
-						height: minHeight,
-					} }
-					minHeight={ COVER_MIN_HEIGHT }
-					enable={ {
-						top: false,
-						right: false,
-						bottom: true,
-						left: false,
-						topRight: false,
-						bottomRight: false,
-						bottomLeft: false,
-						topLeft: false,
-					} }
-					onResizeStop={ ( event, direction, elt, delta ) => {
-						const coverHeight = parseInt( minHeight + delta.height, 10 );
-						this.setState( { coverMinHeight: coverHeight } );
-						setAttributes( {
-							minHeight: coverHeight,
+					onResize={ ( newMinHeight ) => {
```

Comment:
why are we using `onResize`? is it so we update the input live?

Reply:
Exactly to update the input live.

Reply:
could this have performance issues? especially with scaling large images and firing the event too many times? I don't want to premature optimize something and lose a good UX feature (seeing the input being updated) and I guess this won't have any effect since re-resizable is already firing that event, so I guess we can leave it and see if anyone opens an issue regarding this